test/config/k8s/k8s-secret.yml

```diff
@@ -6,4 +6,5 @@ type: Opaque
 stringData:
   conjur-map: |-
     secret: secrets/test_secret
+    var_with_spaces: secrets/var with spaces
   non-conjur-key: some-value
```

Comment:
GENERAL note: you have Geri's commit in your PR. Run `git fetch push ignore` (AKA `git rebase origin/master` git push --force) and it will go away

Reply:
@eladkug ^